src/Microsoft.AspNet.Buffering/HttpBufferingFeature.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Http.Features;
+
+namespace Microsoft.AspNet.Buffering
+{
+    internal class HttpBufferingFeature : IHttpBufferingFeature
```

Comment:
This is the wrong repo/project to add them to. I think they'd go into Microsoft.AspNet.Http.Extensions.